main.go

```diff
@@ -1,174 +1,15 @@
-	prometheus.MustRegister(virtualServerConnsPerSec)
-	prometheus.MustRegister(virtualServerBytesRead)
-	prometheus.MustRegister(virtualServerBytesWritten)
-
-	prometheus.MustRegister(realServerTotalConnections)
-	prometheus.MustRegister(realServerTotalPackets)
-	prometheus.MustRegister(realServerTotalBytes)
-	prometheus.MustRegister(realServerActiveConnections)
-	prometheus.MustRegister(realServerConnsPerSec)
-	prometheus.MustRegister(realServerBytesRead)
-	prometheus.MustRegister(realServerBytesWritten)
-}
-
 func main() {
-	flag.Parse()
-
-	client := kemp.NewClient(kemp.Config{
-		User:     username,
-		Password: password,
-		Endpoint: endpoint,
-		Debug:    debug,
-	})
-
-	go func() {
-		for {
-			statistics, err := client.GetStatistics()
-			if err != nil {
-				fmt.Println(err)
-			}
-
-			connsPerSec.Set(float64(statistics.Totals.ConnectionsPerSec))
-			bytesPerSec.Set(float64(statistics.Totals.BytesPerSec))
-			packetsPerSec.Set(float64(statistics.Totals.PacketsPerSec))
-
-			for _, vs := range statistics.VirtualServers {
-				virtualServerTotalConnections.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.TotalConnections))
-				virtualServerTotalPackets.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.TotalPackets))
-				virtualServerTotalBytes.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.TotalBytes))
-				virtualServerActiveConnections.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.ActiveConnections))
-				virtualServerConnsPerSec.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.ConnectionsPerSec))
-				virtualServerBytesRead.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.BytesRead))
-				virtualServerBytesWritten.WithLabelValues(vs.Address, strconv.Itoa(vs.Port)).Set(float64(vs.BytesWritten))
-			}
-
-			for _, rs := range statistics.RealServers {
-				realServerTotalConnections.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.TotalConnections))
-				realServerTotalPackets.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.TotalPackets))
-				realServerTotalBytes.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.TotalBytes))
-				realServerActiveConnections.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.ActiveConnections))
-				realServerConnsPerSec.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.ConnectionsPerSec))
-				realServerBytesRead.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.BytesRead))
-				realServerBytesWritten.WithLabelValues(rs.Address, strconv.Itoa(rs.Port)).Set(float64(rs.BytesWritten))
-			}
-
-			time.Sleep(WaitTime)
-		}
-	}()
-
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "")
-	})
-	http.Handle("/metrics", prometheus.Handler())
-
-	http.ListenAndServe(":8000", nil)
+	if err := cmd.RootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(-1)
```

Comment:
Huh, that is weird. This is straight from the cobra docs, but I don't like it. I'll change it to `Exit(1)`